backend/validators/grafana_service.py

```
@@ -241,6 +305,198 @@ def sync_network(cls, network):
+                if getattr(operator, field, None) != highest:
+                    Validator.objects.filter(pk=operator.pk).update(**{field: highest})
+                    setattr(operator, field, highest)
+                # (3) Award once the operator (visible) reaches the active target.
+                if (
+                    active
+                    and operator.user_id
+                    and getattr(operator.user, 'visible', False)
+                    and parse_version(highest) >= parse_version(active.version)
+                ):
+                    cls._award_node_upgrade(operator, network, active, now)
+        except Exception as exc:  # pragma: no cover - defensive
```

Ruff (0.15.20)

[warning] Do not catch blind exception: `Exception`

(BLE001)